docs/bugs-features.rst

```diff
@@ -0,0 +1,155 @@
+Bugs and Feature Requests
+=========================
+
+Bugs and feature requests for :term:`pulpcore` are tracked with `Redmine
+<https://pulp.plan.io/projects/pulp/issues/>`_. Please see the :ref:`plugin-table` for trackers for
+each plugin.
+
+.. _issue-writing:
+
+How to file an issue
```

Comment:
This is one of three items in the Table of Contents that I'm just not sure if they should be left alone or capitalized like a title.  I'm leaning towards capitalizing them so that they stand out given that they are headings.